cmd/root.go

```diff
@@ -125,6 +126,45 @@ to assist developers in writing, debugging, and understanding code directly from
 		// Setup the subscriptions, this will send services events to the TUI
 		ch, cancelSubs := setupSubscriptions(app, ctx)
 
+		// Start tmux focus monitoring if in tmux
+		tmuxPane := util.GetTmuxPane()
+		if tmuxPane == "" {
+			logging.Info("Not in tmux session - skipping focus monitoring")
+		} else {
+			logging.Info("Starting tmux focus monitoring", "tmuxPane", tmuxPane)
+			go func() {
+				defer logging.RecoverPanic("tmux-focus-monitor", nil)
+
+				ticker := time.NewTicker(500 * time.Millisecond)
+				defer ticker.Stop()
```

Comment:
i have considered using some other mechanism like registering an event on tmux, and listening to pipes, but maybe a ticker could be the simplest thing to pull off.

we can also guard this with a `OPENCODE_EXPERIMENTAL_TMUX=1` env var for the brave people :)

Reply:
nvm there's a standard escape code